[33m Run failed with command - python3.7 train.py --fp16 --input_dir=../data/tydi/train.h5df use_gpu=True  --output-dir=./test_tipc/output/norm_train_gpus_0_autocast_null --max_steps=200     --batch-size=2 --state_dict_path=null   !  [0m
[33m Run failed with command - python3.7 null   use_gpu=True  !  [0m
[33m Run failed with command - python3.7 tools/export_model.py --model_path=./xnli_outputs/ckpt --save_inference_dir ./xlm_infer    !  [0m
[33m Run failed with command - python3.7 deploy/inference_python/infer.py --model_dir ./xlm_infer --use_gpu=True                 > ./test_tipc/output/python_infer_gpu_usetrt_null_precision_null_batchsize_null.log 2>&1 !  [0m
[33m Run failed with command - python3.7 train.py --fp16 --input_dir=../data/tydi/train.h5df use_gpu=True  --output-dir=./test_tipc/output/norm_train_gpus_0_autocast_null --max_steps=200     --batch-size=2 --state_dict_path=null   !  [0m
[33m Run failed with command - python3.7 null   use_gpu=True  !  [0m
[33m Run failed with command - python3.7 tools/export_model.py --model_path=./xnli_outputs/ckpt --save_inference_dir ./xlm_infer    !  [0m
[33m Run failed with command - python3.7 deploy/inference_python/infer.py --model_dir ./xlm_infer --use_gpu=True                 > ./test_tipc/output/python_infer_gpu_usetrt_null_precision_null_batchsize_null.log 2>&1 !  [0m
[33m Run failed with command - python3.7 train.py --fp16 --input_dir=../data/tydi/train.h5df use_gpu=True  --output-dir=./test_tipc/output/norm_train_gpus_0_autocast_null --max_steps=200     --batch-size=2     !  [0m
[33m Run failed with command - python3.7 null   use_gpu=True  !  [0m
[33m Run failed with command - python3.7 tools/export_model.py --model_path=./xnli_outputs/ckpt --save_inference_dir ./xlm_infer    !  [0m
[33m Run failed with command - python3.7 deploy/inference_python/infer.py --model_dir ./xlm_infer --use_gpu=True                 > ./test_tipc/output/python_infer_gpu_usetrt_null_precision_null_batchsize_null.log 2>&1 !  [0m
[33m Run failed with command - python3.7 train.py --fp16 --input_dir "../data/tydi/train.h5df" use_gpu=True  --output-dir=./test_tipc/output/norm_train_gpus_0_autocast_null --max_steps=200     --batch-size=2     !  [0m
[33m Run failed with command - python3.7 null   use_gpu=True  !  [0m
[33m Run failed with command - python3.7 tools/export_model.py --model_path=./xnli_outputs/ckpt --save_inference_dir ./xlm_infer    !  [0m
[33m Run failed with command - python3.7 deploy/inference_python/infer.py --model_dir ./xlm_infer --use_gpu=True                 > ./test_tipc/output/python_infer_gpu_usetrt_null_precision_null_batchsize_null.log 2>&1 !  [0m
[33m Run failed with command - python3.7 train.py --input_dir "../data/tydi/train.h5df" use_gpu=True  --output-dir=./test_tipc/output/norm_train_gpus_0_autocast_null --max_steps=200     --batch-size=2     !  [0m
[33m Run failed with command - python3.7 null   use_gpu=True  !  [0m
[33m Run failed with command - python3.7 tools/export_model.py --model_path=./xnli_outputs/ckpt --save_inference_dir ./xlm_infer    !  [0m
[33m Run failed with command - python3.7 deploy/inference_python/infer.py --model_dir ./xlm_infer --use_gpu=True                 > ./test_tipc/output/python_infer_gpu_usetrt_null_precision_null_batchsize_null.log 2>&1 !  [0m
[33m Run failed with command - python3.7 train.py --input_dir "../data/tydi/train.h5df" use_gpu=True  --output-dir=./test_tipc/output/norm_train_gpus_0_autocast_null --max_steps=200     --batch-size=2     !  [0m
[33m Run failed with command - python3.7 null   use_gpu=True  !  [0m
[33m Run failed with command - python3.7 tools/export_model.py --model_path=./canine_tydi_qa --save_inference_dir ./canine_infer    !  [0m
[33m Run failed with command - python3.7 deploy/inference_python/infer.py --model_dir ./canine_infer --use_gpu=True                 > ./test_tipc/output/python_infer_gpu_usetrt_null_precision_null_batchsize_null.log 2>&1 !  [0m
